packages/core/content-manager/server/src/history/content-types/history-version.ts

```diff
@@ -0,0 +1,66 @@
+import type { Plugin } from '@strapi/types';
+
+const historyVersion: Plugin.LoadedPlugin['contentTypes'][string] = {
+  schema: {
+    uid: 'plugin::content-manager.history-version',
+    modelName: 'history-version',
+    globalId: 'ContentManagerHistoryVersion',
```

Comment:
Kind of annoying we have to manually define these ones :/

Reply:
Looks like the same type, but I would use this import instead
```
import type { Schema } from "@strapi/types"

Schema.CollectionType
```

Reply:
@Convly Do you know if there is an available type for the User defined schema? One that does not include globalId ?
I have seen i18n defining a type like
` Omit<Schema.ContentType, 'uid' | 'collectionName' | 'globalId' | 'modelName'>`

Maybe the same could be used here? 🤔

Reply:
What's the`globalId` used for and how does it differ from the `uid`? 🤔

Reply:
@Marc-Roig I updated to use Schema.CollectionType.

And yeah I'd be happy to to use a type that doesn't require them. I'll ask others